llvm/lib/Target/EVM/EVMStackLayoutGenerator.cpp

```diff
@@ -77,8 +77,8 @@ findStackTooDeep(Stack const &Source, Stack const &Target) {
           Errors.emplace_back(EVMStackLayoutGenerator::StackTooDeep{
               I - 16, getVariableChoices(take_back(CurrentStack, I + 1))});
       },
-      [&](StackSlot const &Slot) {
-        if (isRematerializable(Slot))
+      [&](StackSlot const *Slot) {
```

Comment:
In other places you moved `const` to the beginning of a declaration. Let's fix this one also.